src/models/document/document.ts

```diff
@@ -20,6 +20,7 @@ export interface IDocumentAddTileOptions {
   title?: string;
   addSidecarNotes?: boolean;
   url?: string;
+  text?: string;
 }
```

Comment:
We could consider changing this to put the title, url, and text in a sub object. So it is clear these fields are being passed to the tool model creator.

Reply:
In the interest of making minimal changes, I'm going to leave this as is.  The text field will be removed in the next commit, so then this block of code is not changed at all.

We discussed the idea that the title could be moved to the wrapper ToolTileModel, so that would then leave the url as the only thing in this sub object is the url. And  I'm hoping that that could be abstracted so it is up to the tile to decide if it can handle dragging in a image instead of the core code hard coding this relationship.

So I guess since these fields might be influx it seems best to just leave them unchanged.

Reply:
I'm going to make a new PR that adds a document with notes about these two improvements so we don't lose track of them.